src/pages/Posts.js

```diff
@@ -4,27 +4,48 @@ import "../styles/Posts.css";
 import Card from "../components/Card";
 
 const Posts = () => {
-  const loggedInUser = JSON.parse(localStorage.getItem("selectedUser"));
   const [posts, setPosts] = React.useState([]);
+  const [allUsers, setAllUsers] = React.useState([]);
+  const allUsersAndPosts = [
+    {
+      name: "",
+      post: "",
+      title: "",
+    },
+  ];
```

Comment:
Is this a list of all users and post, or just a list of posts?

Reply:
list of all users and post